### DISCUSSION TITLE
怎么配置呀？

### DISCUSSION THREAD

**Discussion by Fc8cF** [mikumifa/biliTickerBuy#223] [提问专区]
是在这个页面生成配置吗？
为什么我复制进链接后，点击按钮却什么反应都没有？😭
![2](https://github.com/mikumifa/biliTickerBuy/assets/146006959/ef29a0b3-1371-4ef1-a30b-d142abf84055)

**Comment by Shiliang-vector** [comment]
> 是在这个页面生成配置吗？
> 为什么我复制进链接后，点击按钮却什么反应都没有？😭
> ![2](https://github.com/mikumifa/biliTickerBuy/assets/146006959/ef29a0b3-1371-4ef1-a30b-d142abf84055)
> 

链接格式不对，参照给出的格式只改后面6位id就行

**Reply by Fc8cF** [reply]
我删除了后面的东西，再点击获取按钮还是没有反应TT

**Reply by Shiliang-vector** [reply]
> 我删除了后面的东西，再点击获取按钮还是没有反应TT

![微信截图_20240629134452](https://github.com/mikumifa/biliTickerBuy/assets/137939685/0d24ce33-76f5-405b-a4e4-057dc40b41e8)
wo我这边获取是正常的，你点进登录管理重新登录一下试试
